universal-login-relayer/lib/services/messages/MessageQueueStore.ts

```diff
@@ -39,13 +39,25 @@ export default class MessageQueueStore implements IMessageQueueStore {
   }
 
   async onSuccessRemove (id: string, hash: string) {
+    const messageField = await this.get(id);
+    const messageHash = await calculateMessageHash(bignumberifySignedMessageFields(messageField.message));
+    await this.database('messages')
```

Comment:
That change should be triggered by `MessageQueueService` and executed by `PendingMessagesStore`